test/runtests.jl

```diff
@@ -512,6 +536,16 @@ freqresptest(G, model, tol) = freqresptest(G, model) < tol
         @test compareTFs(G1, Gests)
         
 
+        ## Test na > nb
+        na, nb, inputdelay = 2, 1, 2
+        G1   = tf(0.8, [1,0.5,0.25], 1)
+        u = randn(N)
+        y = lsim(G1, u, t)[1][:]
+        d = iddata(y, u, 1)
+        Gest = arx(d, na, nb, inputdelay = inputdelay)
+        @test G1 ≈ minreal(Gest)
```

Comment:
Why is `minreal` required here? Shouldn't the parameters `na, nb, inputdelay = 2, 1, 2` result in a transfer function of the same order as `G1`?

Reply:
Yes, this minreal() is only a copy-paste artifact and the test should work without it just fine.